composer.json

```diff
@@ -16,9 +16,7 @@
     "propel/propel": "~2.0@dev"
   },
   "require-dev": {
-    "phpunit/phpunit": ">=7.0",
```

Comment:
Can we use PSR4 class loading or is this not possible here?

Reply:
I tried, tests failed with it